src/stores/dashboard/interfaces.ts

```diff
@@ -0,0 +1,36 @@
+import { FindAllDailyOrderResponse, FindAllParams, Order } from '@enigma-laboratory/shared';
+
+export interface ChartUnit {
+  date: Date;
+  value: number;
+}
+
+export type DateFilter = 'lastWeek' | 'lastMonth' | 'custom';
+
+export interface RecentOrder extends Omit<FindAllParams, 'page'>, Required<Pick<FindAllParams, 'page'>> {
+  data: Record<number, Order[]>;
+  count: number;
+  nextPage: boolean;
+  prevPage: boolean;
+}
+
+export interface OrderTimeline extends Omit<FindAllParams, 'page'>, Required<Pick<FindAllParams, 'page'>> {
```

Comment:
I think you should delete **Required<Pick<FindAll Params, 'page'>>**  and  Omit <FindAllParams> instead you should temporarily add a new param to override **page : number**
example :
export interface RecentOrder extends FindAllParams  {
  data: Record<number, Order[]>;
  count: number;
  nextPage: boolean;
  prevPage: boolean;
  page: number;
} 

because page: number overwrite page ?: number